src/pubsub/mod_pubsub_db_mnesia.erl

```diff
@@ -100,23 +101,49 @@ start() ->
                          {attributes, record_info(fields, pubsub_subscription)},
                          {type, set}]),
     mnesia:add_table_copy(pubsub_subscription, node(), disc_copies),
+    CreateSubnodeTableResult = mnesia:create_table(pubsub_subnode,
+                        [{disc_copies, [node()]},
```

Comment:
Would it work for more than one Mongoose node?

Reply:
It changes Mnesia schema, first node that runs it, would do a migration procedure.
Once a table is defined in schema (it's shared for the whole cluster), the only thing nodes can do is to add table copies.